roles/preflight/tasks/test_preflight_check_container_one_image.yml

```diff
@@ -76,7 +76,7 @@
             {{ preflight_image }}
             check container
             {{ current_operator_image }}
-            {% if cert_project_id | default('') | length %}
+            {% if cert_project_id | default('') | length and dci_all_components_are_ga | default(True) %}
```

Comment:
can you change this to
`if (cert_project_id | default('') | length) and (dci_all_components_are_ga | default(True))`
I feel is more readable.

Reply:
Hi @thekad thanks for checking. Tbh, I believe that the current way is more conventional and readable, and I'd prefer to keep it.